README.md

```diff
@@ -1,75 +1,289 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> We welcome new developers to ustilize our Project and build your own apps.
+
+To get a local copy up and running, follow these steps.
 
-To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
+In order to run this project you need to install this project with:
+
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+- Rails
+  ```
+    gem install rails
+  ```
+- Rspec
+  ```
+    bin/rails generate rspec:install
+  ```
 
-### Install
 
 ### Usage
 
+- Open a command line terminal (Git bash preferrably) and navigate to a directory where you would like to save the work folder using cd.
+- Clone the repository with git clone https://github.com/Theophileaseh/Doctor-Appointment-Back-end.git to have a copy of the backend server.
+- To get the curent working tree, git fetch --all, then checkout to "dev" branch.
+- Finally, run the application by doing the following:
+  - navigate to the root directory with a command using terminal
+  - run `bundle install`
+  - run `rake db:create`
+  - run `rake db:migrate`
+  - run `rails s`
+
 ### Run tests
 
+To run tests, run the following command:
+
+- run all test suits
+  ```
+    rspec
+  ```
+- run a specific test file
+  ```
+    rspec spec ./spec/yourtestfile_spec.rb
+  ```
+
+
 ### Deployment
 
+You can deploy this project using Heroku:
+
+```
+  heroku create
+
+```
+```
+  git push heroku development:main
+
+```
+```
+  heroku run rails db:migrate
+
+```
+```
+  heroku open
+
+```
+To see all server logs
+```
+  heroku logs -t
+
+```
 
 
-## Authors
 
-👤 **Author1**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+<!-- AUTHORS -->
 
-👤 **Author2**
+## 👥 Authors <a name="authors"></a>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+> Mention all of the collaborators of this project.
 
-## 🤝 Contributing
+👤 **NWACHAN**
+
+- GitHub: [@Theophileaseh](https://github.com/Theophileaseh)
+
+👤 **CECILIA**
+
+- GitHub: [@c3c1l1a](https://github.com/c3c1l1a)
+
+👤 **MUHAMMAD**
+
+- GitHub: [@ShaheerCH](https://github.com/ShaheerCH)
+
+👤 **NADYA**
+
+- GitHub: [@petushka1](https://github.com/petushka1)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> We are lokking into improving the project and have planed several feature to implement in the future:
+
+- [ ] **[User profile editing]**
+- [ ] **[Doctor Role]**
```

Comment:
- [ ] The square brackets [...] here are just a placeholder and not meant to show on your repo.  Kindly remove them so the document looks better. 👍